templates/aws-stack.yml

```diff
@@ -596,6 +596,14 @@ Parameters:
       - "verify"
     Default: "sign-and-verify"
 
+  PipelineSigningVerificationFailureBehavior:
+    Type: String
+    Description: The access level for the KMS key used to sign and verify pipelines
```

Comment:
I'm not sure if this description matches what the param does?

Reply:
Thanks this is fixed 🤦🏻